flake.nix

```diff
@@ -3,7 +3,7 @@
 
   inputs = {
     flake-utils.url = "github:numtide/flake-utils";
-    nixpkgs.url = "github:nixos/nixpkgs/nixos-22.05";
```

Comment:
Should we use the `haskell-updates` branch instead? I think it's clearer and the exact commit is pinned already in `flake.lock`.

Reply:
I'm happy with that, actually. It's the same commit, that's correct. Let me fix it to `haskell-updates`

Reply:
Cool, fixed now, I'm resolving. Please re-open if you feel like the fix was not what you expected... I hope it's good :) Thanks again!